server/src/main/java/io/druid/guice/http/DruidHttpClientConfig.java

```diff
@@ -35,6 +35,10 @@
   @JsonProperty
   private Period readTimeout = new Period("PT15M");
 
+  @JsonProperty
+  @Min(1)
+  private int numMaxThreads = Math.max(10, (Runtime.getRuntime().availableProcessors() * 17) / 16 + 2) + 30;
```

Comment:
i wanted default value to be same as default for number of jetty threads that this is what it was. see https://github.com/druid-io/druid/blob/master/server/src/main/java/io/druid/server/initialization/ServerConfig.java#L32